stdlib/date/date.flux

```diff
@@ -680,6 +680,31 @@ builtin _truncate : (t: T, unit: duration, location: {zone: string, offset: dura
 // ```
 truncate = (t, unit, location=location) => _truncate(t, unit, location)
 
+// scale will multiply the duration by the given value.
+//
+// ## Parameters
+// - d: Duration to scale.
+// - n: Amount to scale the duration by.
+//
+// ## Examples
+//
+// ### Add n hours to a time
+//
+// ```no_run
+// import "date"
+//
+// n = 5
+// d = date.scale(d: 1h, n: n)
+// date.add(d: d, to: 2022-05-10T00:00:00Z)
+//
+// // Returns 2022-05-10T00:00:00.000000000Z
+// ```
```

Comment:
Can you add an example of how this works with a mixed duration?

Reply:
Done.